dev/AppLifecycle/ActivationRegistrationManager.cpp

```diff
@@ -11,19 +11,15 @@
 #include "ExtensionContract.h"
 #include "EncodedLaunchExecuteCommand.h"
 
-namespace winrt::Microsoft::ApplicationModel::Activation::implementation
+namespace winrt::Microsoft::Windows::AppLifecycle::implementation
 {
-    std::wstring GenerateEncodedLaunchUri(std::wstring const& appUserModelId, std::wstring const& contractId)
-    {
-        Windows::Foundation::Uri uri{ wil::str_printf<std::wstring>(L"%s://%s?%s=%s",
-            c_encodedLaunchSchemeName, appUserModelId.c_str(), c_contractIdKeyName, contractId.c_str()).c_str() };
-        return uri.AbsoluteUri().c_str();
-    }
+    using namespace winrt::Windows::Foundation;
 
     std::wstring GenerateCommandLine(std::wstring const& modulePath)
     {
         std::wstring exePath{ modulePath.empty() ? GetModulePath() : modulePath };
 
+        // Example: C:\some\path\App.exe ----ms-protocol:
         return wil::str_printf<std::wstring>(L"%s %s%s%s", exePath.c_str(), c_argumentPrefix,
             c_protocolArgumentString, c_argumentSuffix);
```

Comment:
Complicated way to say exePath + "----msprotocol:"

I know all the cool kids frown upon #define but that would simplify to exePath.c_str() + C_ARGPREFIX C_ARGSTRING C_ARGSUFFIX

Perhaps 1 typedef constant (PCWSTR) with Prefix+Arg+Suffix to make the preprocessor haters happy and still be efficient and clear?